lib/base-helix-generator.js

```diff
@@ -0,0 +1,29 @@
+const Generator = require('yeoman-generator');
+var globby = require('globby');
+const path = require('path');
+
+const utils = require('./utils.js');
+
+module.exports = class HelixGenerator extends Generator {
```

Comment:
Please, add comments: "why this was added" (workaround for templates in the path)